tests/UI/dashboard.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {expect, test} from '@playwright/test';
+import {BasePage} from "../../Core/Page Object/Base";
+
+let basePage
+test.beforeEach(async ({ page }) => {
+    basePage = new BasePage(page);
+    await basePage.goto('https://reportportal.epam.com/');
+    await basePage.loginPage.loginAsTestUser()
+});
+
+test('Check that user can add new dashboard', async () => {
+    await basePage.dashboardPage.addNewDashboard.click()
```

Comment:
Please consider to unify some actions into methods in the page object, f.e. addNewDashboard() and so on. This would add readability